src/CachePlugin.php

```diff
@@ -72,10 +73,7 @@ public function __construct(CacheItemPoolInterface $pool, StreamFactory $streamF
         $this->streamFactory = $streamFactory;
 
         if (isset($config['respect_cache_headers']) && isset($config['respect_response_cache_directives'])) {
-            throw new \InvalidArgumentException(
-                'You can\'t provide config option "respect_cache_headers" and "respect_response_cache_directives". '.
-                'Use "respect_response_cache_directives" instead.'
-            );
+            throw new \InvalidArgumentException('You can\'t provide config option "respect_cache_headers" and "respect_response_cache_directives". '.'Use "respect_response_cache_directives" instead.');
```

Comment:
did the codestyle checker complain about this before? otherwise lets not touch it.

if codestyle complains, lets remove the concatenation and make it all one string.

Reply:
yes, the code style checker complained about it; should already have been existed before. no magic button (auto format) used.

Reply:
the stupid thing is that it does a string concat on that one line now, for no good reason... can you remove the `'.'` in the middle please?

Reply:
done.